src/views/FileEntity/ExperimentalProcedure/index.tsx

```diff
@@ -0,0 +1,35 @@
+import intl from 'react-intl-universal';
+import { EntityDescriptions } from '@ferlab/ui/core/pages/EntityPage';
+import { IFileEntity } from 'graphql/files/models';
+
+import getExperimentalProcedureItems from '../utils/getExperimentalProcedureItems';
+import { SectionId } from '..';
+
+import ExperimentalProcedureTable from './ExperimentalProcedureTable';
+
+interface OwnProps {
+  file?: IFileEntity;
+  loading: boolean;
+}
+
+const ExperimentalProcedure = ({ file, loading }: OwnProps) => {
+  const nbExperiments = file?.sequencing_experiment?.hits?.edges.length || 0;
+
+  if (nbExperiments > 1) {
```

Comment:
I don't think it would make the code clearer if I were to extract it in a constant. It's really just a check to see if there are multiple results or not, so this number will never change or be used elsewhere. 

2 or more results will display the data in a table component, while 1 or 0 result displays the data in a descriptions component.

So I'll keep it as is for now but I'm open to more feedback.